src/coreclr/tests/src/JIT/opt/InstructionCombining/MulToAdd.cs

```diff
@@ -0,0 +1,133 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+// Test "X * 2" to "X + X"
+
+public class Program
+{
+    private static int resultCode = 100;
+
+    public static int Main(string[] args)
+    {
+        float[] testValues =
+            {
+                0, 0, 0.01f, 1.333f, 1/3.0f, 0.5f, 1, 2, 3, 4,
```

Comment:
> 0, [](start = 18, length = 3)

duplicate input?

Reply:
Ah, it supposed to be `-0` but then I added `testValues.Concat(testValues.Select(v => -v)).ToArray();`